Parsers/Inspiration.js

```diff
@@ -0,0 +1,16 @@
+/*
+activation_example:!inspire
+regex:\b(inspire|inspiration|inspiring|inspired|motivation|motivate|motivated)\b
```

Comment:
Please justify why this is not triggered as a bang command (!example) as these words are not uncommon

Reply:
Thank you for reviewing my contribution. I understand your concern about the potential for over-triggering. My initial reasoning behind using these words was that they are primarily used in discussions where users may be seeking or offering support, so the SlackBot’s response could feel timely and relevant. However, if this approach is too broad, I am happy to adjust the trigger to a bang command ( !inspire | !inspiration)